caldate.h

```diff
@@ -0,0 +1,19 @@
+#ifndef CALDATE_H
+#define CALDATE_H
+
+struct caldate {
+  long year;
+  int month;
+  int day;
+} ;
+
+extern unsigned int caldate_fmt();
```

Comment:
F*ck K&R, we should really add proper C89 signatures for new code, especially like this.